plugins/catalog-backend/scripts/mock-db

```diff
@@ -0,0 +1,12 @@
+curl --location --request POST 'localhost:3003/locations' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "type": "github",
+    "target": "https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/fixtures/one_component.yaml"
+}'
+curl --location --request POST 'localhost:3003/locations' \
```

Comment:
Should probably not add both of these files, don't they even overlap in their contents? Just do the two_components for now, and then we'll try to make an entirely new file that has even more interesting test contents in it some other time.